src/main/java/com/rocketden/main/service/GameService.java

```diff
@@ -15,7 +15,7 @@
 @Service
 public class GameService {
 
-	private static final String START_GAME_SOCKET_PATH = BaseRestController.BASE_SOCKET_URL + "/%s/start-game";
+	private static final String START_GAME_SOCKET_PATH = BaseRestController.BASE_SOCKET_URL + "/%s/subscribe-user";
```

Comment:
I don't think you need this constant; you can just get the one in `WebSocketConfig.java` as `WebSocketConfig.SOCKET_PATH`.